src/main/java/com/starkinfra/StaticBrcode.java

```diff
@@ -0,0 +1,391 @@
+package com.starkinfra;
+
+import com.starkinfra.utils.Generator;
+import com.starkinfra.utils.Resource;
+import com.starkinfra.utils.Rest;
+import com.starkinfra.utils.SubResource;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public final class StaticBrcode extends Resource {
+    /**
+     * StaticBrcode object
+     * <p>
+     * A StaticBrcode stores account information in the form of a PixKey and can be used to create
+     * Pix transactions easily.
+     * When you initialize a StaticBrcode, the entity will not be automatically
+     * created in the Stark Infra API. The 'create' function sends the objects
+     * <p>
+     * Parameters:
+     * name [string]: receiver's name. ex: "Tony Stark"
+     * keyId [string]: receiver's Pixkey id. ex: "+5541999999999"
+     * city [string, default Sao Paulo]: receiver's city name. ex: "Rio de Janeiro"
+     * amount [Long, default 0]: positive integer that represents the amount in cents of the resulting Pix transaction. ex: 1234 (= R$ 12.34)
+     * reconciliationId [string, default "instant"]: id to be used for conciliation of the resulting Pix transaction. ex: "123"
```

Comment:
1) Aqui o default é `""`
2) Adicionar isso:` This id must have up to 25 alphanumeric digits ex: "ah27s53agj6493hjds6836v49"`